src/content/news/2025-02-03-maplibre-newsletter-january-2025/index.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "MapLibre Newsletter January 2025"
+date: "2025-02-03"
+categories: ["newsletter"]
+authors: [bart, birk, ianthetechie, yuri, harel, brandon, josxha, ramyaragupathy]
+draft: false
+---
+
+import { Image } from "astro:assets";
+import gratitude_aws from "./aws-2024-sponsor.png";
+
+As we step into 2025, we want to start with a heartfelt thank you to AWS for renewing their sponsorship for MapLibre. Their ongoing commitment since 2022 has played a vital role in the growth, and strengthening of our global community.
```

Comment:
We should change this a bit to also mention smaller contributors, maybe just by name or for the smallest ones maybe aggregate them somehow?  I do not want us to appear as only receiving sponsorships from the large organizations... I do want to thank AWS for the large portion of the donations though, so they should be front and center.

Reply:
For this edition, I will go with a line acknowledging the contributions in general. In the coming months I will look at improving this content further.